documents/src/pages/elements/list.md

```diff
@@ -50,9 +50,7 @@ const data = [
 el.data = data;
 ```
 
-## Data property interface
-
-The `data` property of the `ef-list` use the [ListData](https://github.com/Refinitiv/refinitiv-ui/blob/develop/packages/elements/src/list/helpers/types.ts) interface for its data items.
+The `data` property of the `ef-list` use the [ListData](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/list/helpers/types.ts) type for its data items. Each of this item defaults to type `ItemData`. It could also be anything extended from [DataItem](./custom-components/utils/data-management#data-item).
```

Comment:
```suggestion
The `data` property of the `ef-list` uses the [ListData](https://github.com/Refinitiv/refinitiv-ui/blob/v6/packages/elements/src/list/helpers/types.ts) type for its data items. Each of this item defaults to type `ItemData`. It could also be anything extended from [DataItem](./custom-components/utils/data-management#data-item).
```